mlflow/langchain/__init__.py

```diff
@@ -680,6 +686,66 @@ def _convert_ndarray_to_list(data):
             f"for model {self.lc_model.__class__.__name__}"
         )
 
+    def predict_stream(
+        self,
+        data: Any,
+        params: Optional[Dict[str, Any]] = None,
+    ) -> Iterator[Union[str, Dict[str, Any]]]:
+        """
+        Args:
+            data: Model input data, only single input is allowed.
+            params: Additional parameters to pass to the model for inference.
+
+                .. Note:: Experimental: This parameter may change or be removed in a future
+                    release without warning.
+        Returns:
+            An iterator of model prediction chunks.
+        """
+        from mlflow.langchain.api_request_parallel_processor import process_stream_request
+
+        # TODO: Q: shall we put `callback_handlers` in params ?
+        if isinstance(data, list):
+            raise MlflowException("Langchain model predict_stream only supports single input.")
+
+        data = _convert_ndarray_to_list(data)
+        return process_stream_request(
+            lc_model=self.lc_model,
+            request_json=data,
+        )
+
+    def _predict_stream_with_callbacks(
+        self,
+        data: Any,
+        params: Optional[Dict[str, Any]] = None,
+        callback_handlers=None,
```

Comment:
```suggestion
        callbacks=None,
```

can we remove `_handlers`? That seems redudant.

Reply:
Got it, `callback_handlers` sounds good.